presentation-api/controlling-ua/defaultRequest_success-manual.html

```diff
@@ -27,35 +33,30 @@
     // 'default request' Test - BEGIN
     // ------------------------------
     async_test(function(t) {
-        var request = new PresentationRequest(presentationUrls);
-        if('defaultRequest' in navigator.presentation) {
-            navigator.presentation.defaultRequest = request;
-
-            if('onconnectionavailable' in request) {
-                request.onconnectionavailable = t.step_func_done(function (evt) {
-                    var connection = evt.connection;
-                    notice.parentNode.removeChild(notice);
-
-                    assert_equals(connection.state, 'connecting', 'The initial state of the presentation connection is "connecting".');
-                    assert_true(!connection.id, 'The connection ID is set.');
-                    assert_true(typeof connection.id === 'string', 'The connection ID is a string.');
-                    assert_true(connection instanceof PresentationConnection, 'The connection is an instance of PresentationConnection.');
-                });
-                button.onclick = t.step_func_done(function() {
-                    button.disabled = true;
-                    assert_unreached('This browser does not support defaultRequest.');
-                });
-            }
-            else {
-                notice.parentNode.removeChild(notice);
-                assert_unreached('This browser does not support connectionavailable event.');
-            }
-        }
-        else {
+        // clean up the instruction notice when the test ends
+        t.add_cleanup(function() {
             notice.parentNode.removeChild(notice);
+        });
+        // set an event handler to make the test fail when the button is clicked
+        button.onclick = t.step_func_done(function() {
             assert_unreached('This browser does not support defaultRequest.');
-        }
-    }, '[Optional] The presentation was started successfully.');
+        });
+        // check if defaultRequest is an instance of PresentationRequest or null
+        var defaultRequest = navigator.presentation.defaultRequest;
+        assert_true(defaultRequest instanceof PresentationRequest || defaultRequest === null,
+            'navigator.presentation.defaultRequest must be an instance of PresentationRequest or null.');
```

Comment:
[The current spec](http://w3c.github.io/presentation-api/#dom-presentation-defaultrequest) says:

> "The `defaultRequest` attribute MUST return the default presentation request if any, `null` otherwise."

However, its default value is not clearly mentioned yet. Therefore, this assertion checks if the default value is either an instance of PresentationRequest or null, just in case.